examples/vector-db/README.md

```diff
@@ -0,0 +1,26 @@
+# NuLang Translator
```

Comment:
We've typically had this match the example folder name a bit more closely. Maybe we call this "Retrieval Augmented Generation "and can include details in the scenario in the text.

Was also chatting with Evan about this and i wonder if we should just make this RAG over the GenSX docs so it's a more business-y scenario. If you wanted, i'd be happy to put together a scraper side of this so the example could essentially allow a user to paste in a url to their website and get RAG over their website...or we could just leverage the monorepo and point to the gensx docs